content/en/blog/2024/authz-policy-with-kyverno/index.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Policy based authorization - Using Kyverno Authz Server
+description: Delegate authorization decisions logic to Kyverno Authz Server, leveraging Kyverno authorization policies based on CEL bringing safety, flexibility, and unprecedented performance.
+publishdate: 2024-11-20
+attribution: "Charles-Edouard Brétéché (Nirmata)"
+keywords: [istio,kyverno,policy,platform,authorization]
+---
+
+A couple of weeks ago, the [Can Your Platform Do Policy? Accelerate Teams With Platform L7 Policy Functionality](../l7-policy-with-opa) blog post was published.
+
+This blog post is a follow-up article, demonstrating the same capacity with the Kyverno Authz Server instead of OPA.
+
+Today, we're going to dive into how Istio and the Kyverno Authz Server can be used together to enforce Layer 7 policies in your platform.
+We'll show you how to get started with a simple example.
+You will come to see how this combination is a solid option to deliver policy quickly and transparently to application team everywhere in the business, while also providing the data the security teams need for audit and compliance.
+
+## Try it out
+
+When integrated with Istio, the Kyverno Authz Server can be used to enforce fine-grained access control policies for microservices.
+
+This guide shows how to enforce access control policies for a simple microservices application.
+
+### Prerequisites
+
+- A Kubernetes cluster with Istio installed.
+- The `istioctl` command-line tool installed.
+
+Install Istio and configure your [mesh options](/docs/reference/config/istio.mesh.v1alpha1/) to enable Kyverno:
+
+{{< text bash >}}
+$ istioctl install -y -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+    accessLogFormat: |
+      [KYVERNO DEMO] my-new-dynamic-metadata: '%DYNAMIC_METADATA(envoy.filters.http.ext_authz)%'
+    extensionProviders:
+    - name: kyverno-authz-server.local
```

Comment:
I find it confusing that this has the appearance of a DNS name; would just `kyverno-auth-server` make more sense throughout?